Dfe.Data.SearchPrototype/Web/Tests/ViewTests/SearchPagePresentationTests.cs

```diff
@@ -30,16 +35,16 @@ public async Task Search_WithNoEstablishmentResultsAndNoFacets_ShowsNoResultsTex
             .ReturnsAsync(useCaseResponse);
 
         // act
-        var resultsPage = await _context.OpenAsync($"{homeUri}?searchKeyword=anything");
-
-        // assert
-        resultsPage.QuerySelector(HomePage.SearchNoResultText.Criteria)!
-            .TextContent.Should().Contain("Sorry no results found please amend your search criteria");
-        resultsPage.QuerySelector(HomePage.SearchResultsNumber.Criteria)!
-            .Should().BeNull();
-        resultsPage.GetElementById("filters-container").Should().BeNull();
+        const string searchKeywordQueryWithNoResults = $"?searchKeyword=anything";
+        Response response = await _client.GetHttpResponseAsync(searchKeywordQueryWithNoResults);

```

Comment:
Left leaks `QuerySelector`,  AngleSharp, `.Criteria`  WebDriver `.TextContext` AngleSharp
Unable to reuse the expectation of no search results being displayed like in `HomePageAssertionExtensions` across tests